MapboxVisionFleet/VisionFleetManager/VisionFleetManager.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import MapboxVision
+import MapboxVisionFleetNative
+import MapboxVisionSafety
+
+public final class VisionFleetManager {
+
+    public weak var delegate: VisionFleetManagerDelegate?
+
+    public static func create(visionManager: VisionManagerProtocol, visionSafetyManager: VisionSafetyManager) -> VisionFleetManager {
+        let manager = VisionFleetManager()
```

Comment:
shouldn't we create native fleet manager? and make a proxy for native events?

Reply:
Yes, but not in this PR. This one is just for the module and basic class skeletons. API will be refined and implemented later.